scripts/components/OpenSearch-Dashboards/install.sh

```diff
@@ -66,11 +70,22 @@ fi
 [ -z "$SNAPSHOT" ] && SNAPSHOT="false"
 [ -z "$PLATFORM" ] && PLATFORM=$(uname -s | awk '{print tolower($0)}')
 [ -z "$ARCHITECTURE" ] && ARCHITECTURE=`uname -m`
+[ -z "$DISTRIBUTION" ] && DISTRIBUTION="tar"
 
 ## Setup default config
-(
+if [ "$DISTRIBUTION" = "tar" ]; then
+(   
     DIR="$(dirname "$0")"
     echo $DIR
     cd $DIR
     cp ../../../config/opensearch_dashboards.yml "$OUTPUT/config/"
 )
+elif [ "$DISTRIBUTION" = "rpm" ]; then
+(   
+    DIR="$(dirname "$0")"
+    echo $DIR
+    cd $DIR
```

Comment:
Why is this needed? I don't expect scripts to change current directory. 

If there's a good reason, the `DIR` part is common and should be taken out of the `if`.

Reply:
I forget to make it common as I was copy paste during testing, will change.

Reply:
fixed.